recipe/meta.yaml

```diff
@@ -11,6 +11,7 @@ source:
     sha256: 71c8ba7777ffdca37638bc796524d42c51006fad385967fa140235b60aac0252
     patches:
       - conda-build.patch
+      - fix_clang_errors.patch
```

Comment:
Would suggest sending these upstream if not done already. Should allow these to be dropped in the future

Reply:
@jakirkham I am in the process of getting ready to upstream these. :) Hopefully will go smoothly.

Reply:
Yes! So our plan is upstream the cmake patch, we will need patches for 1.1.2 as well. The clang patch is a back port from upstream, so when we build the next version we will not need it.